src/lib/Serialize.hs

```diff
@@ -90,24 +90,26 @@ getTableElements tab = do
 -- This doesn't handle parentheses well. TODO: treat it more like PrettyPrec
 prettyVal :: (MonadIO1 m, EnvReader m, Fallible1 m) => Val n -> m n (Doc ann)
 prettyVal val = case val of
+  -- Pretty-print strings
+  DataCon "AsList" _ [Word8Ty] _ [_, tab@(TabVal _ _)] -> do
+    atoms <- getTableElements tab
```

Comment:
Can you please use `getDexString` here? It's _a lot_ faster

Reply:
Good idea